modules/bucksenseBidAdapter.js

```diff
@@ -0,0 +1,171 @@
+import { registerBidder } from 'src/adapters/bidderFactory';
+import { BANNER } from 'src/mediaTypes';
+
+const WHOIS = 'BKSHBID-008';
+const BIDDER_CODE = 'bucksense';
+const URL = 'https://prebid.bksn.se:445/prebid';
+var bksdebug = false;
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER],
+
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {object} bid The bid to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+  */
+  isBidRequestValid: function (bid) {
+    if (isNaN(bid)) {
+      return false;
+    }
+    if (bid.params.debug) {
+      bksdebug = bid.params.debug;
+    }
+    if (bksdebug) console.log(WHOIS + ' isBidRequestValid() - INPUT bid:', bid);
+    return Boolean(bid.bidId && bid.params && !isNaN(bid.params.placementId));
+  },
+
+  /**
+    * Make a server request from the list of BidRequests.
+    *
+    * @param {BidRequest[]} validBidRequests A non-empty list of valid bid requests that should be sent to the Server.
+    * @return ServerRequest Info describing the request to the server.
+   */
+  buildRequests: function (validBidRequests, bidderRequest) {
+    if (bksdebug) console.log(WHOIS + ' buildRequests() - INPUT validBidRequests:', validBidRequests, 'INPUT bidderRequest:', bidderRequest);
+    let requests = [];
+
+    const len = validBidRequests.length;
+    for (let i = 0; i < len; i++) {
+      var bid = validBidRequests[i];
+
+      requests.push({
+        method: 'POST',
+        url: URL,
+        data: {
+          'pub_id': location.host,
+          'pl_id': bid.params.placementId,
+          'sys_href': encodeURI(location.href),
```

Comment:
If you haven't already looked at it, there are a set of data points you can use in your adapter to get information on the webpage loading the adapter.  Please see the link below and consider using them if they would fit your needs:
http://prebid.org/dev-docs/bidder-adaptor.html#referrers

Reply:
Thanks for point to this but in this release we would like to keep using the location object if it is not a problem